tests/libumaptest/mmap.cpp

```diff
@@ -24,6 +24,7 @@ Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 #include <string.h>
 #include "umap.h"
 #include "umaptest.h"
+#include <vector>
```

Comment:
These changes should be reverted and calls should continue to use "C" interface that was there previously.